darwin/torch/transforms.py

```diff
@@ -8,8 +8,7 @@
 
 from darwin.torch.utils import convert_segmentation_to_mask, flatten_masks_by_category
 
-
-TargetKey = Union["boxes", "labels", "mask", "masks", "image_id", "area", "iscrowd"]
+TargetKey = Literal["boxes", "labels", "mask", "masks", "image_id", "area", "iscrowd"]
```

Comment:
Nice addition.  Always love a Literal

Reply:
Annoyingly I think it's caused other issues to crop up elsewhere. Which is half the problem changing some of these. It's like whack-a-mole